### ISSUE TITLE
Frontend stackdump  -0 - Call to a member function getMappedItems() on null

### ISSUE THREAD

**Issue by jobrusche** [JoomGalleryfriends/JG4-dev#75]
Menu-item in frontend = List of images
Klik on image-name causes stackdump : 0 - Call to a member function getMappedItems() on null
![Album-image-list](https://user-images.githubusercontent.com/117929258/216391647-a8b097c5-4991-4064-977d-655c2811b6e4.png)

![Categor-list-image](https://user-images.githubusercontent.com/117929258/216391755-e09ee9a8-22c9-4ec8-95f1-2a8bb3fd07b2.png)

Environment Wampserver / Joomla 4.2.7 / PHP 8.018

**Comment by MrMusic** [comment]
@jobrusche Thank you for your hint.
At the moment, only the backend is under development. It will take a while until the frontend is usable.

**Comment by Elfangor93** [comment]
Yes, currently we focus development to the administration area. As soon as the backend is running smoothly with the most important functions we will switch to frontend development.
